packages/react-native-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/GutenbergProps.kt

```diff
@@ -10,7 +10,8 @@ data class GutenbergProps(
     val editorTheme: Bundle?,
     val translations: Bundle,
     val isDarkMode: Boolean,
-    val htmlModeEnabled: Boolean
+    val htmlModeEnabled: Boolean,
+    val isModalLayoutPickerEnabled: Boolean
```

Comment:
I was thinking of adding a default value to the new parameter in order to be able to merge this early without breaking the WP-Android code. Something like the following:

```
data class GutenbergProps @JvmOverloads constructor(
    val enableMentions: Boolean,
    val enableUnsupportedBlockEditor: Boolean,
    val localeSlug: String,
    val postType: String,
    val editorTheme: Bundle?,
    val translations: Bundle,
    val isDarkMode: Boolean,
    val htmlModeEnabled: Boolean
    val htmlModeEnabled: Boolean,
    val isModalLayoutPickerEnabled: Boolean = false
) {
```

Wdyt @mchowning , @chipsnyder ?

Reply:
That seems like a good plan to me.